aviary/examples/external_subsystems/engine_NPSS/run_simple_engine.py

```diff
@@ -0,0 +1,47 @@
+"""
+Run the a mission with a simple external component that computes the wing
+and horizontal tail mass.
```

Comment:
copy-paste from simple_mass?

Reply:
Good catch on this one.   Updated to an appropriate comment then uploaded.